cypress/support/commands.ts

```diff
@@ -0,0 +1,26 @@
+Cypress.Commands.add("signup", () => {
+    cy.visit("/");
+
+    const { email, password } = Cypress.env();
+
+    cy.get("pl-app").find("pl-start").find("pl-login-signup").find("pl-input#emailInput").find("input").type(email);
```

Comment:
Once I have the whole login passing, I'll spend some time investigating a less verbose way of doing this. My initial tries (just `.get('pl-app pl-start pl-login-signup...')`) didn't work, unfortunately. I also had spent quite some time properly adding `.shadow()` in the right places before realizing `includeShadowDom` was a thing, now.